tests/unit/s2n_x509_validator_test.c

```diff
@@ -472,10 +471,10 @@ int main(int argc, char **argv) {
         struct s2n_pkey public_key_out;
         EXPECT_SUCCESS(s2n_pkey_zero_init(&public_key_out));
         s2n_pkey_type pkey_type;
-        EXPECT_EQUAL(S2N_CERT_ERR_UNTRUSTED,
-                     s2n_x509_validator_validate_cert_chain(&validator, connection, chain_data, chain_len, &pkey_type, &public_key_out));
-        s2n_stuffer_free(&chain_stuffer);
+        EXPECT_ERROR_WITH_ERRNO(s2n_x509_validator_validate_cert_chain(&validator, connection, chain_data, chain_len,
+                &pkey_type, &public_key_out), S2N_ERR_CERT_UNTRUSTED);
 
```

Comment:
This test is labeled "test expired certificate fails as untrusted". Shouldn't this be S2N_ERR_CERT_EXPIRED?

Reply:
Yes, probably. I made a commit to pull the expired error out of the X509_verify call: 80b79b6fddcc1348493056405e0d7e8ae5a785bc

I looked at the other openssl error codes for this function and didn't find anything that seemed relevant, but if you think there's more errors we can pull out of this that would be useful let me know.